packages/expo-dev-menu/src/DevMenu.ts

```diff
@@ -11,3 +14,38 @@ export function openProfile(): void {
 export function openSettings(): void {
   ExpoDevMenu.openSettings();
 }
+
+let hasRegisteredCallbackListener = false;
+
+function registerCallbackListener() {
+  if (!hasRegisteredCallbackListener) {
+    DeviceEventEmitter.addListener('registeredCallbackFired', (name: string) => {
+      hasRegisteredCallbackListener = true;
+      const handler = handlers.get(name);
+
+      if (handler != null) {
+        handler();
+      }
+    });
+  }
+}
+
+registerCallbackListener();
+
+let handlers = new Map<string, () => void>();
+
+export async function registerDevMenuItems(items: ExpoDevMenuItem[]) {
+  if (!__DEV__) {
+    return Promise.resolve();
```

Comment:
i think you can just `return;` here, async will take care of promisifying if needed